### ISSUE TITLE
Game crashes upon mounting.

### ISSUE THREAD

**Issue by alphosra** [darkevilmac/archimedes-ships-plus#349]
**_- Mod version_**
6.340

**_- Minecraft version_**
1.12.2

_**- Forge version**_
14.23.5.2854

**_- Crash report/log_**
https://gist.github.com/alphosra/528de4554f0ab2f5cdea3a5efc07a676

**Comment by xXDeonoXx** [comment]
same here, if i assemble the ship, close the menu then try to use the ship helm again, the game crashs.

**Comment by beli-coder** [comment]
Yup, same here. Except, this happened on my own server. If I assemble, then its fine. But when I try to mount, it crashes my game instantly. My computer is very beefy, so I know that the amount of entities did not crash it. Can anyone help?

**Comment by valzzu** [comment]
same here. In singleplayer works fine but when in multiplayer game every client connected crashes but not server. Any help?
